prac_07/guitar.py

```diff
@@ -0,0 +1,26 @@
+CURRENT_YEAR = 2022
+VINTAGE_AGE = 50
+
+
+class Guitar:
+    """Not sure"""
```

Comment:
We want two doc strings for a class, one at the start of the file and then one for each class in the file. In this case "Guitar class" for both I think should have been ok.